### ISSUE TITLE
When using lightly, dolphin switches back to the old style space information style. 

### ISSUE THREAD

**Issue by linusrg1** [boehs/Lightly#1]
At least from the photos of lightly, it should be using the modern thin style space meter instead of the older, larger style one. 
![Screenshot_20221103_184447](https://user-images.githubusercontent.com/67347319/199849255-9729a713-a54a-4ae1-866b-5d8ff3f1c5b1.png)

Sys info: 
Operating System: openSUSE Tumbleweed 20221024
KDE Plasma Version: 5.26.1
KDE Frameworks Version: 5.99.0
Qt Version: 5.15.6
Kernel Version: 6.0.3-1-default (64-bit)
Graphics Platform: Wayland
Processors: 16 × 12th Gen Intel® Core™ i5-1240P
Memory: 15.3 GiB of RAM
Graphics Processor: Mesa Intel® Graphics
Manufacturer: Framework
Product Name: Laptop (12th Gen Intel Core)
System Version: A4

I also have these other theme elements applied: 
SierraBreezeEnhanced window decorations
A weird conglomeration of two different plasma styles

If I apply the stock breeze application style the issue goes away. 

If you need any more info, just tell me how to get it and I'd be happy to provide you with it.

**Comment by boehs** [comment]
Tracked in https://github.com/Luwx/Lightly/issues/69

**Comment by boehs** [comment]
Resolution for openSUSE https://github.com/Luwx/Lightly/issues/69#issuecomment-1114110684

I can research how much of this is lightly vs openSUSE, but my inclination is it's openSUSE

**Comment by linusrg1** [comment]
Thanks, it works now. Installing those extra packages didn't work at first, but I think that was because I also updated opensuse at the same time, and opensuse replaced the version of lightly with the one from the sputnik repo, and because of this, installing those extra packages didn't fix the issue. But I uninstalled the one from the sputnik repo, rebooted my machine for safe measure, and then installed the version that I built from the latest source code located here, and now it works!
